models/gpg_key.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/openpgp"
+	"golang.org/x/crypto/openpgp/packet"
+
+	"code.gitea.io/gitea/modules/log"
+	"github.com/go-xorm/xorm"
+)
```

Comment:
please following the style guide. https://github.com/go-gitea/gitea/blob/master/CONTRIBUTING.md#styleguide